Narrowing conversion on windows

I get 
```
libpointmatcher\pointmatcher\DataPointsFilters\MaxPointCount.cpp(73,42): error C2398: Element '1': conversion from 'size_t' to 'unsigned int' requires a  a narrowing conversion
```
When building with MSVC 2019 on Windows.

--- a/pointmatcher/DataPointsFilters/MaxPointCount.cpp
+++ b/pointmatcher/DataPointsFilters/MaxPointCount.cpp
@@ -70,7 +70,7 @@ void MaxPointCountDataPointsFilter<T>::inPlaceFilter(DataPoints& cloud)
 
 	if (maxCount <= N)
 	{
-		std::minstd_rand randomNumberGenerator{seed};
+		std::minstd_rand randomNumberGenerator(static_cast<std::uint_fast32_t>(seed));
 		std::uniform_real_distribution<float> distribution{0, 1};
 
 		for (size_t j{0u}; j < maxCount; ++j)

